html/achievements.html

```diff
@@ -0,0 +1,85 @@
+<div class="divisiontop light"></div>
+<div class="division dark">
+    <h1 class="division_description" style="grid-column: 1 / 13;">Erhaltene Erfolge (11)</h1>
+    <div class="box light" style="grid-column: span 3;">
+        <img class="achievement_icon" src="images/dummy_achievement_icon.png">
+    </div>
+    <div class="box light" style="grid-column: span 3;">
+        <img class="achievement_icon" src="images/dummy_achievement_icon.png">
+    </div>
+    <div class="box light" style="grid-column: span 3;">
+        <img class="achievement_icon" src="images/dummy_achievement_icon.png">
+    </div>
+    <div class="box light" style="grid-column: span 3;">
+        <img class="achievement_icon" src="images/dummy_achievement_icon.png">
+    </div>
+    <div class="box light" style="grid-column: span 3;">
+        <img class="achievement_icon" src="images/dummy_achievement_icon.png">
+    </div>
+    <div class="box light" style="grid-column: span 3;">
+        <img class="achievement_icon" src="images/dummy_achievement_icon.png">
+    </div>
+    <div class="box light" style="grid-column: span 3;">
+        <img class="achievement_icon" src="images/dummy_achievement_icon.png">
+    </div>
+    <div class="box light" style="grid-column: span 3;">
+        <img class="achievement_icon" src="images/dummy_achievement_icon.png">
+    </div>
+    <div class="box light" style="grid-column: span 3;">
+        <img class="achievement_icon" src="images/dummy_achievement_icon.png">
+    </div>
+    <div class="box light" style="grid-column: span 3;">
+        <img class="achievement_icon" src="images/dummy_achievement_icon.png">
+    </div>
+    <div class="box light" style="grid-column: span 3;">
+        <img class="achievement_icon" src="images/dummy_achievement_icon.png">
+    </div>
+</div>
+<div class="division light">
+    <h1 class="division_description" style="grid-column: 1 / 13;">Offene Erfolge (15)</h1>
+    <div class="box dark" style="grid-column: span 3;">
+        <img class="achievement_icon" src="images/dummy_achievement_icon_dark.png">
```

Comment:
The dark version of `dummy_achievement_icon` should use the same image file with some css to make it darker (semi-transparent grey div in front of the image?). This way we don't have to make two images for each achievement.